joomla/joomla-cms
# [4.0] update the custom elements

Pull Request for Issue # .

### Summary of Changes
- update webcomponentjs (the polyfills) to the latest version (2.0.3)
- remove the hacked tabs custom element from media/system/webcomponents
- Fix the two known bugs for tabs (appendChild instead of append and tabs.id instead of tabs.hash)

### Testing Instructions
Apply patch (probably won't work with patchtester)
Edit an article
Check the tabs functionality
Save Check the alert element
exit and re-edit the same article, observe that the tabstate is kept

### Expected result



### Actual result



### Documentation Changes Required

@wilsonge
